From b9af77c4326862474b303b05ee9c2ad37d2862ec Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Tue, 17 Nov 2015 19:38:45 +0200
Subject: [PATCH 0508/1240] armada3700: tim: Fix the DDR init procedure in NTIM

- Fix the DDR initiaization procedure according to latest
  debugger script values tested on DB @ 600 MHz

Change-Id: Ifbf24b69e4c13a7594c9eb90a1067ab3a974f6d0
Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/24964
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
---
 tools/wtp/u-boot-ntim.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tools/wtp/u-boot-ntim.txt b/tools/wtp/u-boot-ntim.txt
index 654728c..5fe0c15 100644
--- a/tools/wtp/u-boot-ntim.txt
+++ b/tools/wtp/u-boot-ntim.txt
@@ -69,16 +69,16 @@ WRITE: 0xC0014008 0x00404500
 WRITE: 0xC0002000 0x00010000
 WRITE: 0xC0002004 0x00000000
 WRITE: 0xC0000340 0x0f0f0fef
-WRITE: 0xC0000344 0xaa
+WRITE: 0xC0000344 0x100000aa
 WRITE: 0xC0000310 0x200000
 WRITE: 0xC0000304 0x0
 WRITE: 0xC0000308 0x0
 WRITE: 0xC0000200 0xe0001
 WRITE: 0xC0000204 0x0
-WRITE: 0xC0000220 0x14000632
+WRITE: 0xC0000220 0x16020532
 WRITE: 0xC0000044 0x30200
 WRITE: 0xC00002c0 0x6000
-WRITE: 0xC00002c4 0x20
+WRITE: 0xC00002c4 0x00100020
 WRITE: 0xC0000058 0x143f
 WRITE: 0xC0000048 0x1
 WRITE: 0xC0000180 0x00010200
-- 
1.9.1

